java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3069,88 +3079,188 @@ void testWindowingLead() {
         try (Scalar two = Scalar.fromInt(2);
              Scalar one = Scalar.fromInt(1);
              WindowOptions options = windowBuilder.window(two, one).build();
-             WindowOptions options1 = windowBuilder.window(two, one).build()) {
-          try (Table windowAggResults = sorted.groupBy(0, 1)
-              .aggregateWindows(Aggregation
-                  .lead(0)
-                  .onColumn(3)
-                  .overWindow(options));
-               Table decWindowAggResults = decSorted.groupBy(0, 4)
-                   .aggregateWindows(Aggregation
-                       .lead(0)
-                       .onColumn(6)
-                       .overWindow(options1));
-               ColumnVector expectAggResult = ColumnVector.fromBoxedInts(7, 5, 1, 9, 7, 9, 8, 2, 8, 0, 6, 6);
-               ColumnVector decExpectAggResult = ColumnVector.decimalFromLongs(-2, 7, 5, 1, 9, 7, 9, 8, 2, 8, 0, 6, 6)) {
-            assertColumnsAreEqual(expectAggResult, windowAggResults.getColumn(0));
-            assertColumnsAreEqual(decExpectAggResult, decWindowAggResults.getColumn(0));
-          }
+             Table windowAggResults = sorted.groupBy(0, 1)
+                 .aggregateWindows(Aggregation
+                     .lead(0)
+                     .onColumn(3)
```

Comment:
It is hard to read/follow the example. (Maybe I'm not used to these examples any more.)

It might be good add a comment to indicate which column is being aggregated.
```suggestion
                     .onColumn(3) // Int agg column.
```

Reply:
Done